setup.py

```diff
@@ -12,6 +12,13 @@ def readme():
             return f.read()
 
 
+requires = ['boto>=2.33.0',]
+
+
+ if sys.version_info[:2] == (2, 6):
```

Comment:
Sorry for my late response, I weren't notified via email for this PR.

I doubt this is enough as this is probably also required for Python < 2.7 **and** < 3.2.